src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -181,7 +193,7 @@ define([
                         }
                     }),
                     error: lang.hitch(this, function(error) {
-                        logger.error(this.id + ": An error ocurred while executing microflow: ", error);
+                        mx.ui.error("An error ocurred while executing microflow" + error.message);
                     })
                 };
                 if (!mx.version || mx.version && parseInt(mx.version.split(".")[0]) < 7) {
```

Comment:
verison stuff, can be removed as this widget is no longer will run < 7.13.1

Reply:
done